vernac/vernacprop.ml

```diff
@@ -0,0 +1,53 @@
+(************************************************************************)
```

Comment:
I see what you mean, however vernac_classifier currently needs access to the VCS whereas the simpler classifier does not.